TFproviderTest.yml

```diff
@@ -39,14 +40,19 @@ pipelines:
             - echo "Run Helm installation script" && ./get_helm.sh && rm get_helm.sh
             - helm version
             - apt update
-            - export GO_VERSION="1.17.8"
             - export GO_ARCH="amd64"
+            - rm -rf *.tar.gz go
             - curl -O -L "https://golang.org/dl/go${GO_VERSION}.linux-${GO_ARCH}.tar.gz"
             - ls -l
             - tar -xf "go${GO_VERSION}.linux-${GO_ARCH}.tar.gz"
             - sudo chown -R root:root ./go
+            - >-
+              if [ -d /usr/local/go ]; then
```

Comment:
Not the correct way to do this

`command -v go` - this is the portable way to check if an app is installed on the bin path - And if you're going to do that `rm -fR` you need to put `rm -fR /usr/local/go || true` because if for some reason it isn't there rm will return status non-0 and your shell will error

Reply:
You know, more simply you can just do `rm -fR /usr/local/go || true` you don't even need the `if`